### ISSUE TITLE
Unable to resolve module `AccessibilityInfo`

### ISSUE THREAD

**Issue by Asinox** [Yamilquery/react-native-timekeeper#5]
There is a way to fix this issue?

`error: bundling failed: Error: Unable to resolve module AccessibilityInfo from node_modules/react-native-timekeeper/node_modules/react-native/Libraries/react-native/react-native-implementation.js: AccessibilityInfo could not be found within the project.`

react-native-cli: 2.0.1
react-native: 0.61.5

Thanks!

**Comment by JohnBerd** [comment]
"react": "16.3.1"
"react-native": "0.55.4"
 would solve the problem
